src/applications/vaos/new-appointment/components/VAFacilityPage/FacilitiesRadioWidget.jsx

```diff
@@ -47,19 +57,25 @@ export default function FacilitiesRadioWidget({
         ) {
           distance = legacyVAR?.distanceFromCurrentLocation;
         }
+        const facilityPosition = facilityIndex + 1;
 
         return (
-          <div className="form-radio-buttons" key={option.value}>
+          <div
+            className="form-radio-buttons"
+            key={facility.value}
+            aria-atomic="true"
+            aria-live="assertive"
```

Comment:
Same question as before, not sure why we're adding these aria-live attributes

Reply:
Deleted